permits/templates/permits/permit_request_detail.html

```diff
@@ -224,11 +216,25 @@ <h2>{% trans "Données de base" %}</h2>
       </a>
     </dd>
   {% endif %}
-  {% if config.ENABLE_GEOCALENDAR %}
-  <dt>{% trans "Publié" %}</dt>
-  <dd>
-    {{permit_request.is_public|yesno:"Oui,Non" }}
-  </dd>
+  {% if history %}
+    <div id="history_timeline">
+      <h4><b>{% trans "Suivi" %}</b></h4>
```

Comment:
```suggestion
      <h4>{% trans "Suivi" %}</h4>
```

Keep the same style for the titles

Reply:
And maybe use term "Historique" ?